packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -97,7 +97,7 @@ const DEFAULT_COLUMN_PROPERTIES: Partial<StarkTableColumnProperties> = {
 		])
 	]
 })
-export class StarkTableComponent extends AbstractStarkUiComponent implements OnInit, AfterViewInit, OnChanges, OnDestroy {
+export class StarkTableComponent<T extends Object> extends AbstractStarkUiComponent implements OnInit, AfterViewInit, OnChanges, OnDestroy {
```

Comment:
Could you replace the `Object` type by `object` in order to keep the same behavior as before?


```suggestion
export class StarkTableComponent<T extends object> extends AbstractStarkUiComponent implements OnInit, AfterViewInit, OnChanges, OnDestroy {
```